pkgs/by-name/at/attract-mode/package.nix

```diff
@@ -0,0 +1,77 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchpatch2,
+  pkg-config,
+  expat,
+  ffmpeg_7,
+  freetype,
+  libarchive,
+  libjpeg,
+  libGLU,
+  sfml,
+  zlib,
+  openal,
+  fontconfig,
+  darwin,
+}:
+
+stdenv.mkDerivation {
+  pname = "attract-mode";
+  version = "2.7.0-unstable-2024-08-02";
+
+  src = fetchFromGitHub {
+    owner = "mickelson";
+    repo = "attract";
+    rev = "6ed3a1e32a519608c0b495295cc4c18ceea6b461";
+    hash = "";
```

Comment:
empty hash 👀

Reply:
Uhh sorry I feel like I’ve been at the bottom of an abandoned mineshaft for a year after https://github.com/NixOS/nixpkgs/pull/332035 and apparently I’m frazzled 🫠

Fixed!